atlasdb-cli/src/test/java/com/palantir/atlasdb/cli/command/TestTimestampCommand.java

```diff
@@ -90,13 +92,18 @@ public TestAtlasDbServices connect(ServicesConfigModule servicesConfigModule) {
     @Parameterized.Parameter(value = 1)
     public boolean isToFile;
 
+    @Parameterized.Parameter(value = 2)
+    public boolean isFileWithMissingDir;
```

Comment:
We should write proper tests for this. There shouldn't really be any parameters in this test file but... oh well.

Try adding a test that checks what happens if the folder cannot be written, etc.